data_validation_tests/test.yaml

```diff
@@ -0,0 +1,14 @@
+test:
```

Comment:
Should rename the file to sharepoint_preprocessor_test.yaml

Additonially @ecoavu should get dotenv in before we commit anymore changes with credentials

Reply:
Renamed